demo/simple.html

```diff
@@ -15,6 +15,7 @@
 
 				body {
 					margin: 0;
+					margin-bottom: 500px;
```

Comment:
This is a little hacky, but I think what's happening is: galen looks at initially how tall the window is and then ignores whether it grows later based on dynamic content. So the bottom of the dynamically created tables were getting clipped in the image captures. Creating space for them like this solves it.

Reply:
It could be that the screen capture happens before the dynamic table renders.

Reply:
Except that adding this margin to the bottom fixes it such that it captures both dynamically rendered tables perfectly.